mastercraft.cmd

```diff
@@ -2049,6 +2116,7 @@ lack.material:
                if "%order.type" = "genich" then var order.num 11
                if "%order.type" = "ojhenik" then var order.num 12
                if "%order.type" = "red.flowers" then var order.num 13
+			   if "%order.type" = "flowers" then var order.num 13

```

Comment:
I added support for "flowers" instead of "red.flowers". Basically the adverb for flowers changes once it has been combined, and it creates ALL KINDS OF PROBLEMS. (I think red flowers becomes dried flowers or something)